lib/rules/no-dom-import.ts

```diff
@@ -40,11 +37,10 @@ export default createTestingLibraryRule<Options, MessageIds>({
 
   create(context, [framework], helpers) {
     function report(
-      node: TSESTree.ImportDeclaration | TSESTree.Identifier,
+      node: TSESTree.ImportDeclaration | TSESTree.CallExpression,
       moduleName: string
     ) {
       if (framework) {
-        const isRequire = isIdentifier(node) && node.name === 'require';
```

Comment:
small doubt here: I guess some of the rules had support for require, but that only applied to a few. Are we dropping the support for `require` in all of them?
I am not opposed to it, I'm wondering if we should document it somewhere? (not sure if the docs for this rule and others showed it as part of the valid examples)

Reply:
yup in this case the `require` examples are in the docs https://github.com/testing-library/eslint-plugin-testing-library/blob/master/docs/rules/no-dom-import.md

Reply:
oh ups nevermind, I saw the helpers consider require as well, great! 🚀  you're one step ahead

Reply:
Sorry, I should have pointed this with a comment on this PR. But yest, I basically extracted the imported module checks from this rule (which were the best ones) into detection helpers. So now all refactores rules automatically getting the imports from either `import` or `require` syntax ✨